src/pages/Facility/settings/organizations/FacilityOrganizationIndex.tsx

```diff
@@ -85,43 +92,216 @@ export default function FacilityOrganizationIndex({
     >
-      <div className="flex justify-center md:justify-end mt-2 mb-4">
-        <CreateFacilityOrganizationSheet facilityId={facilityId} />
+      <h2 className="mb-4 text-[#000000]">{t("departments")}</h2>
+      <div className="flex justify-between items-center mb-4 gap-2 flex-wrap">
+        <div className="w-60">
+          <Input
+            className="px-2 placeholder:text-xs placeholder:text-[#4B5563]"
+            placeholder={t("filter_by_department_or_team_name")}
+          ></Input>
+        </div>
+        <div className="flex md:justify-end">
+          <CreateFacilityOrganizationSheet facilityId={facilityId} />
+        </div>
       </div>
-      <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4 pb-4">
-        {data.results.map((org: FacilityOrganization) => (
-          <Card key={org.id} className="relative group">
-            <CardHeader>
-              <div className="flex items-start justify-between">
-                <CardTitle className="flex items-center gap-2">
-                  {org.name}
-                </CardTitle>
-                <CardDescription>{org.org_type}</CardDescription>
-              </div>
-            </CardHeader>
-
-            <CardFooter>
-              <Button variant="outline" asChild className="w-full">
-                <Link
-                  href={`/departments/${org.id}`}
-                  className="flex items-center justify-center gap-2"
-                >
-                  {t("view_details")}
-                  <CareIcon icon="l-arrow-right" className="h-4 w-4" />
-                </Link>
-              </Button>
-            </CardFooter>
-          </Card>
-        ))}
+      <div className="flex-col sm:flex-row items-center flex gap-3 text-[#1E3A8A] text-sm border-2 rounded-lg border-[#BFDBFE] bg-[#EFF6FF] p-4 mb-4">
+        <div className="p-2 bg-[#DBEAFE] rounded-sm">
+          <CareIcon icon="l-info-circle" className="h-6 w-6 text-[#1E40AF]" />
+        </div>
+        <div className="">
+          <p className="">
+            {t("click")}{" "}
+            <span className="font-semibold">{t("add_department_or_team")}</span>{" "}
+            {t("to_create_a_new_one")}
+          </p>
+          <p className="">
+            {t("click")}{" "}
+            <span className="font-semibold">{t("see_details")}</span>{" "}
+            {t(
+              "to_open_manage_users_or_create_more_departments_teams_within_it",
+            )}
+          </p>
+        </div>
       </div>
+      <Table className="border rounded-lg w-full overflow-hidden">
+        <TableHeader>
+          <TableRow>
+            <TableHead className="w-[80%] border text-[#374151]  bg-[#f3f4f6]">
+              {t("name")}
+            </TableHead>
+            <TableHead className="bg-[#f3f4f6] text-[#374151]">
+              {t("category")}
+            </TableHead>
+          </TableRow>
+        </TableHeader>
+        <TableBody>
+          {data.results
```

Comment:
For readability - do the filtering earlier and set to a diff variable and use that here.